cmake/modules/FindPySide.cmake

```diff
@@ -51,10 +51,12 @@ if (pySideImportResult EQUAL 1 OR PYSIDE_USE_PYSIDE)
     endif()
 endif()
 
+# If nothing is found, the result will be <VAR>-NOTFOUND.
 find_program(PYSIDEUICBINARY NAMES ${pySideUIC} HINTS ${PYSIDE_BIN_DIR})
 
 if (pySideImportResult)
-    if (EXISTS ${PYSIDEUICBINARY})
+    # False if the constant ends in the suffix -NOTFOUND.
+    if (PYSIDEUICBINARY)
```

Comment:
Left a similar comment in build_usd.py -- should we add `uic` to the `pySideUIC` list above at line 37? (GitHub annoyingly doesn't let me comment on lines unaffected by the PR...)

Reply:
Yes, I think adding uic to the list of executables to search for above would be good.

Reply:
Added 'uic' to list of pyside executables.